lib/fastlane_core/project.rb

```diff
@@ -151,6 +151,20 @@ def configurations
       results
     end
 
+    def default_app_identifier
+      scheme = schemes.first if is_workspace
+      default_build_settings(key: "PRODUCT_BUNDLE_IDENTIFIER", scheme: scheme)
+    end
+
+    def default_app_name
+      if is_workspace
+        scheme = schemes.first
```

Comment:
Is it a save assumption to just select the first scheme? Doesn't it make more sense to ask the user if there are more than one scheme?